util/uvmdvgen/checklist.md.tpl

```diff
@@ -26,15 +26,15 @@ RTL           | [FUNC_IMPLEMENTED][]           | Not Started |
 RTL           | [ASSERT_KNOWN_ADDED][]         | Not Started |
 Code Quality  | [LINT_SETUP][]                 | Not Started |
 
-[SPEC_COMPLETE]:              {{<relref "/doc/project/checklist.md#spec_complete" >}}
-[CSR_DEFINED]:                {{<relref "/doc/project/checklist.md#csr_defined" >}}
-[CLKRST_CONNECTED]:           {{<relref "/doc/project/checklist.md#clkrst_connected" >}}
-[IP_TOP]:                     {{<relref "/doc/project/checklist.md#ip_top" >}}
-[IP_INSTANTIABLE]:            {{<relref "/doc/project/checklist.md#ip_instantiable" >}}
-[PHYSICAL_MACROS_DEFINED_80]: {{<relref "/doc/project/checklist.md#physical_macros_defined_80" >}}
-[FUNC_IMPLEMENTED]:           {{<relref "/doc/project/checklist.md#func_implemented" >}}
-[ASSERT_KNOWN_ADDED]:         {{<relref "/doc/project/checklist.md#assert_known_added" >}}
-[LINT_SETUP]:                 {{<relref "/doc/project/checklist.md#lint_setup" >}}
+[SPEC_COMPLETE]:              ../../../../doc/project_governance/checklist/README.md#spec_complete
+[CSR_DEFINED]:                ../../../../doc/project_governance/checklist/README.md#csr_defined
+[CLKRST_CONNECTED]:           ../../../../doc/project_governance/checklist/README.md#clkrst_connected
+[IP_TOP]:                     ../../../../doc/project_governance/checklist/README.md#ip_top
+[IP_INSTANTIABLE]:            ../../../../doc/project_governance/checklist/README.md#ip_instantiable
+[PHYSICAL_MACROS_DEFINED_80]: ../../../../doc/project_governance/checklist/README.md#physical_macros_defined_80
+[FUNC_IMPLEMENTED]:           ../../../../doc/project_governance/checklist/README.md#func_implemented
+[ASSERT_KNOWN_ADDED]:         ../../../../doc/project_governance/checklist/README.md#assert_known_added
+[LINT_SETUP]:                 ../../../../doc/project_governance/checklist/README.md#lint_setup
```

Comment:
Are ip_autogen blocks' checklists generated from this template as well (e.g. [ip_autogen/rv_plic/doc/checklist.md](https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/ip_autogen/rv_plic/doc/checklist.md))?

If so, these relative links may sadly not always be correct.

Reply:
I see, so we may have to pass the relative path to the OT root directory into the template.

I don't know if `uvmdvgen` is invoked by another tool or manually for IPs in `ip_autogen`, but in any case I think at least `uvmdvgen` has to be changed to output correct relative links

Reply:
I think a similar problem / situation exists with `util/make_new_dif.py` and `doc/project_governance/checklist/sw_checklist.md.tpl`

Reply:
Should we fix `util/uvmdvgen/checklist.md.tpl` and `doc/project_governance/checklist/sw_checklist.md.tpl` in this PR, so that *most* cases get fixed, and defer changes to `uvmdvgen` and `make_new_dif.py` to a separate PR (tracked by an issue for now, because I at least don't have time to work on it atm)?

Reply:
Or we could fall back to using absolute links (relative to the project root e.g. `/doc/project_governance/checklist/README.md`).

I'm cool with either.

Reply:
Ah, of course, that's much simpler for project-relative links. For block/IP-relative links, I think relative links should work fine